Makefile

```diff
@@ -385,8 +392,9 @@ capd-test: ## Run default e2e capd test locally
 	$(MAKE) local-e2e LOCAL_E2E_TESTS=$(DOCKER_E2E_TEST)
 
 .PHONY: docker-e2e-test
-docker-e2e-test: e2e ## Run docker integration test in new ec2 instance
+docker-e2e-test: release-cluster-controller e2e ## Run docker integration test in new ec2 instance
 	scripts/e2e_test_docker.sh $(DOCKER_E2E_TEST) $(BRANCH_NAME)
+	aws ecr-public batch-delete-image --repository-name eks-anywhere-cluster-controller --image-ids=imageTag=$(PULL_PULL_SHA).$(PULL_BASE_REF) --region us-east-1
```

Comment:
can we make this always run, regardless of failure?

Reply:
I'll put it in the trap